plutus-benchmark/nofib/src/PlutusBenchmark/NoFib/Queens.hs

```diff
@@ -203,6 +203,7 @@ instance TxPrelude.Eq Assign
     where (a := b) == (a' := b') = a==a' && b==b'
 instance TxPrelude.Ord Assign
     where (a := b) < (a' := b') = (a<a') || (a==a' && b < b')
+          (a := b) <= (a' := b') = (a<a') || (a==a' && b <= b')
```

Comment:
@kwxm do you remember anything about the `nofib` implementation? Is it fine to make this change here? Should we just disable the warning not allowing us to make this change?

Reply:
OK, whatever, @aleeusgr could you please just add

```
{-# OPTIONS_GHC -Wno-missing-methods #-}
```

to the top of the file and that's it?

Reply:
... and keep the original 

```
(a := b) < (a' := b') = (a<a') || (a==a' && b < b')
```

definition.